app/js/components/maintenance/maintenanceCategorySelect/maintenanceCategorySelect.directive.js

```diff
@@ -38,8 +37,13 @@ function maintenanceCategorySelectCtrl($rootScope,
         tag: gettextCatalog.getString('Maintenance: Future plans')
     }];
 
+    $scope.selected = find($scope.options, {id: defaultCategory});
+    $scope.plans = MaintenanceService.plans;
+    setUrl();
+
```

Comment:
lines 40 and 42 should probably be replaced with the $scope.select function to be consistent on category change. Also, we should be using $scope.category as the default category.